docs/commands/intro-to-command-blocks.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Intro to Command Blocks
+category: General
+mention:
+ - BedrockCommands
+ - zheaEvyline
+nav_order: 1
+tags:
+ - info
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+Command Blocks are special blocks in Minecraft. The same commands (cheats) you type in chat can be run automatically using command blocks and it can be reused without needing to type all over again.
+
+They can only be placed or destroyed by a player with the Operator permission level in gamemode Creative.
+
+## Obtaining
+
+1. Open your world settings.
+2. Under Cheats, toggle "Activate Cheats" setting ON.
+3. Run `/give @s command_block` command in chat.
+
+## Command Block UI
+
+![commandBlockUI](/assets/images/commands/commandBlockUI.png)
+
+## Command Block Types
+
+![impulseCommandBlock](/assets/images/commands/impulseCommandBlock.png) **Impulse** runs the command __once__ each time it is powered.
+
+![chainCommandBlock](/assets/images/commands/chainCommandBlock.png) **Chain** runs the command in a sequence, ie. only after the previous command block it is connecting from was run.
+
+![repeatingCommandBlock](/assets/images/commands/repeatingCommandBlock.png) **Repeat** runs the command every game tick. There are approx 20 ticks per second. A delay can be applied to adjust how often the command is executed (explained [below](/intro-to-command-blocks.html#command-block-tick-delay).)
```

Comment:
`approx` -> `approximately`
Omit the explain below or take it out of the parenthesis